gapir/cc/context.cpp

```diff
@@ -792,4 +797,18 @@ bool Context::stopTimer(Stack* stack, bool pushReturn) {
   return false;
 }
 
+bool Context::sendNotificationData(Stack* stack) {
+  GAPID_WARNING("Sending notification data via gPRC service.");
```

Comment:
Should this be sent as a warning?

Reply:
This will spam warnings, which we should avoid.